docs/pages/en-us/Overview.md

```diff
@@ -36,6 +36,9 @@ The batch worker takes batches of work, recursively breaks them down into ever s
 
 The Job Service is particularly beneficial to the batch worker because it monitors the progress of the batch as a whole. Even if the size of the batch is not known in advance, using the Job Service makes it possible to monitor the overall progress of the entire batch, as well as making other operations available, such as the capability to cancel the batch.
 
+### Dependent Jobs
+The Job Service has the ability to execute jobs in an order defined by a job having a dependency on another job.  The Job Service will not execute a job until all the jobs it is dependent upon have completed.  Once all dependent jobs have completed the Job Service will automatically execute the job which was previously waiting upon dependent jobs.
```

Comment:
The Job Tracking Worker is the component that automatically starts a job when its prerequisites have been completed.

Reply:
@CJComac That is an irrelevant implementation detail (from a user perspective).

Reply:
"The Job Service has the ability to execute jobs in an order defined by a job having a dependency on another job.": This doesn't seem very clear to me.

Reply:
Updated:
`Once all dependent jobs have completed the job tracking worker will automatically forward the job which was previously waiting upon dependent jobs.`

Reply:
Leaving as it is i.e. `Once all dependent jobs have completed the Job Service will automatically execute the job which was previously waiting upon dependent jobs.`